groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -138,15 +135,15 @@ BSLS_IDENT("$Id: $")
 #if defined(BSLS_COMPILERFEATURES_SUPPORT_REF_QUALIFIERS)
 #define BSLS_KEYWORD_LVREF_QUAL &
 #define BSLS_KEYWORD_RVREF_QUAL &&
-    // These macros represent ref-qualifiers on member functions in C++11
+// These macros represent ref-qualifiers on member functions in C++11
 #else
 #define BSLS_KEYWORD_LVREF_QUAL
-//#define BSLS_KEYWORD_RVREF_QUAL
-    // In C++03, there is no notion of rvalues, and all member functions are
-    // effectively lvalue qualified.  'BSLS_KEYWORD_RVREF_QUAL' is deliberately
-    // not defined in C++03, as any reasonable implementation of a member
-    // function with this qualifier, i.e. one that takes advantage of the fact
-    // it has been invoked on an rvalue, has no meaning in C++03.
+//#define BSLS_KEYWORD_RVREF_QUAL In C++03, there is no notion of rvalues, and
+// all member functions are effectively lvalue qualified.
+// 'BSLS_KEYWORD_RVREF_QUAL' is deliberately not defined in C++03, as any
+// reasonable implementation of a member function with this qualifier, i.e. one
+// that takes advantage of the fact it has been invoked on an rvalue, has no
+// meaning in C++03.
```

Comment:
Unfortunately, running bde-format on the entire file doesn't always yield good results - like here for example.  I think the most efficient way of making this better would likely be:

1. Revert to the previous version.
2. Integrate bde-format in your editor (we have instructions for multiple editors including vim/emacs/VS Code and some more - let me know if you need them).
3. Go through the file "section-by-section" formatting and inspecting them.
4. Formatting just the C++11 variadic case and re-running the script to generate the C++03 versions.

It's more manual work than it would be ideally, but it's still better than doing everything completely manually.

Also, I would really appreciate if you could squash the commits that came after my last review - it's quite difficult to see the diff from what I've already seen (unless there's a better way to do that and I'm just unaware of it).

Reply:
I use eclipse. If you have instructions for that, it would be useful. In the mean time, eclipse does offer a side by side view of current changes and I used that. 
Commits squashed. Let me know if this conversation can be resolved.

Reply:
We don't seem to have instructions for eclipse, but since bde-format is just clang-format in disguise (with slight modifications), I think with [this](https://marketplace.eclipse.org/content/cppstyle) and [setup](https://github.com/wangzw/cppstyle) plugin in proper environment (i.e., if the plugin just looks for clang-format, it should find bde-format executable), it should work just fine.

Reply:
Thanks for the hints. I'll play around with it on Monday and ping you if i get stuck.
Was I supposed to run bde-format with  `--style BDE` ?

Reply:
I believe so.

Reply:
ack. Re-ran with --style BDE, squashed the commit, and pushed it.